packages/app/components/itemtable/itemTable.tsx

```diff
@@ -70,25 +73,81 @@ export const ItemsTable = ({
   const TableItem = ({ itemData }: TableItemProps) => {
     const { name, weight, category, quantity, unit, _id, type } = itemData;
 
+    const [isEditModalVisible, setEditModalVisible] = useState(undefined);
+    const [isDeleteModalVisible, setDeleteModalVisible] = useState(undefined);
+
+    const toggleEdit = () => {
+      setEditModalVisible(!isEditModalVisible);
+    };
+
+    const toggleDelete = () => {
+      setDeleteModalVisible(!isDeleteModalVisible);
+    };
+
     const rowData = [
-      name,
-      `${formatNumber(weight)} ${unit}`,
-      quantity,
-      `${category?.name || type}`,
-      <EditPackItemModal>
-        <AddItem
-          _id={_id}
-          isEdit={true}
-          isItemPage
-          initialData={itemData}
-          editAsDuplicate={false}
-          setPage={setPage}
-          page={page}
-        />
-      </EditPackItemModal>,
-      <DeletePackItemModal itemId={_id} />,
+      <Text style={styles.rowText} key="name">
+        {name}
+      </Text>,
+      <Text
+        style={styles.rowText}
+        key="weight"
+      >{`${formatNumber(weight)} ${unit}`}</Text>,
+      <Text style={styles.rowText} key="quantity">
+        {quantity}
+      </Text>,
+      <Text
+        style={styles.rowText}
+        key="category"
+      >{`${category?.name || type}`}</Text>,
+      <View
+        key="viewContainer"
+        style={{ flex: 1, justifyContent: 'center', alignItems: 'flex-end' }}
+      >
+        <DropdownMenu.Root>
```

Comment:
We should pull this into another component to its easier to follow. same file is fine. 

also if we are going to use DropDownMenu directly we should add some basic styles to web version so it looks decent. the docs have a good example using dripsy we could apply to tamagui/vanilla stylesheet.